ui/src/api/jq/schema.ts

```diff
@@ -2,18 +2,33 @@ import { z } from "zod";
 /**
  * example
   {
-    "query":  ".foo[1]",
-    "data":  "eyJmb28iOiBbeyJuYW1lIjoiSlNPTiIsICJnb29kIjp0cnVlfSwgeyJuYW1lIjoiWE1MIiwgImdvb2QiOmZhbHNlfV19",
-    "results":  [
-      "{\n  \"good\": false,\n  \"name\": \"XML\"\n}"
-    ]
+    "data": {
+      "jx": null,
+      "data": "e30=",
+      "output": [
+        "e30="
+      ],
+      "logs": ""
+    }
   }
  */
 
 export const JqQueryResult = z.object({
-  query: z.string(),
-  data: z.string(),
-  results: z.array(z.string()),
+  data: z.object({
+    jx: z.string().nullable(),
+    data: z.string(),
+    output: z.array(z.string()),
+    logs: z.string(),
+  }),
 });
 
 export type JqQueryResultType = z.infer<typeof JqQueryResult>;
+
+export const ExecuteJqueryPayloadSchema = z.object({
```

Comment:
```suggestion
export const ExecuteJqQueryPayloadSchema = z.object({
```
The previously defined types have `jqQuery`, I think we should follow that pattern to avoid confusion with JavaScript JQuery.